### PR TITLE
i18n(es): Translated `from-eleventy.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `from-eleventy.mdx` to spanish 😁

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template blog
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+Trae tus archivos Markdown existentes (o MDX, con nuestra integración opcional) como contenido para [crear páginas Markdown o MDX](/es/guides/markdown-content/).
+
+Tu proyecto Eleventy te permitió usar una variedad de lenguajes plantilla para construir tu sitio. En un proyecto Astro, tu plantilla de página se logrará principalmente con componentes de Astro, que se pueden usar como elementos de IU, diseños e incluso páginas completas. Puedes explorar [la sintaxis de componentes de Astro](/es/core-concepts/astro-components/) para ver cómo plantillar en Astro usando componentes.
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx:62]:
```suggestion
Tu proyecto Eleventy te permitió usar una variedad de lenguajes de plantillas para construir tu sitio. En un proyecto Astro, la plantilla de tus página se logrará principalmente con componentes de Astro, los cuales pueden ser utilizados como elementos UI, diseños e incluso páginas completas. Puedes explorar [la sintaxis de componentes de Astro](/es/core-concepts/astro-components/) para ver cómo se puede crear plantillas en Astro mediante el uso de componentes.
```

File: src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+- Astro usa una [carpeta `src/`](/es/core-concepts/project-structure/#src) para todos los archivos, incluidos los metadatos del sitio, que están disponibles para consultas y procesamiento durante la construcción del sitio. Dentro de esto hay una [carpeta especial `src/pages/` para el enrutamiento basado en archivos](/es/core-concepts/astro-pages/).
+
+- Astro usa una [carpeta `public/` para activos estáticos](/es/core-concepts/project-structure/#public) que no necesitan ser procesados ni transformados durante la construcción.
+
+- En Eleventy, el empaquetado de CSS, JavaScript y otros activos debe configurarse manualmente. [Astro lo maneja por ti de forma predeterminada](/es/concepts/why-astro/#muchas-características-pero-flexible).
+
+## Migrar de Eleventy a Astro
+
+Para convertir un blog de Eleventy a Astro, comienza con nuestra plantilla de inicio de tema de blog, o explora más temas de blog de la comunidad en nuestra [mostrador de temas](https://astro.build/themes/).
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx:38]:
```suggestion
Para convertir un blog de Eleventy a Astro, comienza con nuestra plantilla de inicio de tema de blog, o explora más temas de blog de la comunidad en nuestro [mostrador de temas](https://astro.build/themes/).
```

File: src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+- Astro y Eleventy construyen [aplicaciones de múltiples páginas](/es/concepts/mpa-vs-spa/).
+
+## Principales diferencias entre Eleventy (11ty) y Astro
+
+Cuando reconstruyas tu sitio de Eleventy (11ty) en Astro, notarás algunas diferencias importantes:
+
+- Eleventy admite una variedad de lenguajes de plantillas. Astro admite [la inclusión de componentes de varios marcos de trabajo de JavaScript populares (por ejemplo, React, Svelte, Vue, Solid)](/es/core-concepts/framework-components/), pero utiliza [diseños, páginas y componentes de Astro](/es/core-concepts/astro-components/) para la mayoría de las plantillas de páginas.
+
+- Astro usa una [carpeta `src/`](/es/core-concepts/project-structure/#src) para todos los archivos, incluidos los metadatos del sitio, que están disponibles para consultas y procesamiento durante la construcción del sitio. Dentro de esto hay una [carpeta especial `src/pages/` para el enrutamiento basado en archivos](/es/core-concepts/astro-pages/).
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx:30]:
```suggestion
- Astro usa una [carpeta `src/`](/es/core-concepts/project-structure/#src) para todos los archivos, incluyendo los metadatos del sitio, que están disponibles para consultar y procesar durante la construcción del sitio. Dentro de este directorio se encuentra una [carpeta especial `src/pages/` para el enrutamiento basado en archivos](/es/core-concepts/astro-pages/).
```

File: src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+- Astro y Eleventy te permiten usar un [CMS headless, APIs o archivos Markdown para datos](/es/guides/data-fetching/). Puedes continuar usando tu sistema de [autoría de contenido](/es/guides/content/) preferido, y podrás mantener tu contenido existente.
+
+- Astro y Eleventy construyen [aplicaciones de múltiples páginas](/es/concepts/mpa-vs-spa/).
+
+## Principales diferencias entre Eleventy (11ty) y Astro
+
+Cuando reconstruyas tu sitio de Eleventy (11ty) en Astro, notarás algunas diferencias importantes:
+
+- Eleventy admite una variedad de lenguajes de plantillas. Astro admite [la inclusión de componentes de varios marcos de trabajo de JavaScript populares (por ejemplo, React, Svelte, Vue, Solid)](/es/core-concepts/framework-components/), pero utiliza [diseños, páginas y componentes de Astro](/es/core-concepts/astro-components/) para la mayoría de las plantillas de páginas.
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx:28]:
```suggestion
- Eleventy admite una variedad de lenguajes de plantillas. Astro admite [la inclusión de componentes de varios frameworks populares de JavaScript (por ejemplo, React, Svelte, Vue, Solid)](/es/core-concepts/framework-components/), pero utiliza [los diseños, páginas y componentes de Astro](/es/core-concepts/astro-components/) para la mayoría de las plantillas de páginas.
```

File: src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+[Eleventy](https://11ty.dev) es un generador de sitios estáticos de código abierto que funciona con varios lenguajes de plantillas.
+
+## Principales similitudes entre Eleventy (11ty) y Astro
+
+Eleventy (11ty) y Astro comparten algunas similitudes que te ayudarán a migrar tu proyecto:
+
+- Tanto Astro como Eleventy son generadores de sitios modernos basados en JavaScript (Jamstack).
+
+- Astro y Eleventy te permiten usar un [CMS headless, APIs o archivos Markdown para datos](/es/guides/data-fetching/). Puedes continuar usando tu sistema de [autoría de contenido](/es/guides/content/) preferido, y podrás mantener tu contenido existente.
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx:20]:
```suggestion
- Astro y Eleventy te permiten usar un [CMS headless, APIs o archivos Markdown para los datos](/es/guides/data-fetching/). Puedes continuar usando tu sistema de [autoría de contenido](/es/guides/content/) preferido, y podrás mantener tu contenido existente.
```

File: src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx [hunk 6]
```diff
@@ -0,0 +1,68 @@
+---
+title: Migrating from Eleventy
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-eleventy.mdx:2]:
```suggestion
title: Migrando desde Eleventy
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🎉, te dejo un par de sugerencias

**Comment by dreyfus92** [comment]
Gracias por el review @Waxer59 🤓

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great job folks

**Pull request merged**